howso/direct/client.py

```diff
@@ -2926,9 +2926,6 @@ def react(  # noqa: C901
                 raise HowsoError("`num_cases_to_generate` must be an integer "
                                  "greater than 0.")
             total_size = num_cases_to_generate
-            if total_size > self._react_generative_batch_threshold:
```

Comment:
is this variable used anywhere else? should it be cleaned out everywhere as well?

Reply:
it is. 2 spots, once where the same limit is imposed on react_series, and the other to indicate that the react should be batched. So I am not removing it, as I think the react_series limit is still applicable and the indicator that the react should be batched is still relevant.